package.json

```diff
@@ -136,6 +135,9 @@
     "react-dom": "^0.14 || ^15.5.4 || ^16.1.1",
     "react-with-direction": "^1.3.0"
   },
+  "optionalDependencies": {
```

Comment:
ah, i misunderstood :-/ this would make it an optional runtime dep of react-dates - there’s no optional dev deps. The only way to make this work is to either a) have it be an optional dep of another package that is a dev dep here, or b) don’t put it in package.json at all and manually install it in travis.yml, or c) find an alternative package

Reply:
Looks like that didn't work...I'll probably remove it from package.json then and just install it when it's needed.